src/otk/command.py

```diff
@@ -42,12 +42,16 @@ def _process(arguments: argparse.Namespace, dry_run: bool) -> int:
 
     if arguments.input is None:
         path = pathlib.Path(f"/proc/self/fd/{sys.stdin.fileno()}")
+        # non-existing fake file to get the paths right
+        cwd = pathlib.Path.cwd() / "stdin"
+        state = State(path=cwd)
     else:
         path = pathlib.Path(arguments.input)
+        cwd = path
```

Comment:
I think using `cwd` is misleading here, in my mind `cwd` stands for "current-working-directory` but here we assign a filename, so /home/mvogt/foo.otk would be `cwd` - this seems to have potential for confusion when causually reading the code.

But of course it's tricky to find a good name because we already have `path` and the fact that we now have/need a second path is confusing. This indicates that something in our abstractions is off, probably worth adding a TODO at least that having two paths to workaround weirdness in the stdin handling is something we should look at again.

Reply:
In the stdin case it is really only used as current working directory for includes but I'll come up with a better/different name